docs/guides/fetch-data/01-read-profile-data.md

```diff
@@ -0,0 +1,520 @@
+const { ERC725 } = require('@erc725/erc725.js');
+require('isomorphic-fetch');
+
+// Parameters for ERC725 Instance
+const erc725schema = [
+  {
+    name: 'SupportedStandards:ERC725Account',
+    key: '0xeafec4d89fa9619884b6b89135626455000000000000000000000000afdeb5d6',
+    keyType: 'Mapping',
+    valueContent: '0xafdeb5d6',
+    valueType: 'bytes',
+  },
+  {
+    name: 'LSP3Profile',
+    key: '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
+    keyType: 'Singleton',
+    valueContent: 'JSONURL',
+    valueType: 'bytes',
+  },
+  {
+    name: 'LSP1UniversalReceiverDelegate',
+    key: '0x0cfc51aec37c55a4d0b1a65c6255c4bf2fbdf6277f3cc0730c45b828b6db8b47',
+    keyType: 'Singleton',
+    valueContent: 'Address',
+    valueType: 'address',
+  },
+];
+
+const provider = new Web3.providers.HttpProvider(
+  'https://rpc.l14.lukso.network',
+);
+
+const config = {
+  ipfsGateway: 'https://ipfs.lukso.network/ipfs/',
+};
+
+/*
+ * Try fetching the @param's Universal Provile data
+ * @returns: string JSON or custom error
+ */
+async function getProfile(address) {
+  if (isAddrValid(address)) {
+    try {
+      const erc725 = new ERC725(erc725schema, address, provider, config);
+      return await erc725.fetchData();
+    } catch (err) {
+      return console.log('This is not an ERC725 Contract');
+    }
+  } else {
+    return console.log('This is not an blockchain address');
+  }
+}
+
+// Debug
+getProfile(sampleProfileAddress).then((profileData) =>
+  console.log(JSON.stringify(profileData, undefined, 2)),
+);
+```
+
+If everything went fine, we now have the profile's dataset as a universal JSON structure that looks like this:
+
+<details>
+    <summary>Show JSON responce</summary>
+
+```json
+{
+  "SupportedStandards:ERC725Account": "0xafdeb5d6",
+  "LSP3Profile": {
+    "LSP3Profile": {
+      "name": "...",
+      "links": [
+        {
+          "title": "... ",
+          "url": "..."
+        },
+        ...
+      ],
+      "description": "...",
+      "profileImage": [
+        {
+          "width": 1512,
+          "height": 1998,
+          "hashFunction": "keccak256(bytes)",
+          "hash": "0x...",
+          "url": "..."
+        },
+        ...
+      ],
+      "backgroundImage": [
+        {
+          "width": 1512,
+          "height": 1998,
+          "hashFunction": "keccak256(bytes)",
+          "hash": "0x...",
+          "url": "..."
+        },
+        ...
+      ],
+      "tags": ["sample profile"]
+    }
+  },
+  "LSP1UniversalReceiverDelegate": "0x..."
+}
+```
+
+</details>
+
+## Step 3 - Read the Dataset
+
+With the JSON response, we can fetch all sorts of data:
+
+- Profile metadata
+- Pictures and their properties
+- Receiver address containing asset data
+
+Just instantiate variables for the information you would like to process. All demo fetch functions provided with this guide will come with a debug parameter. You can set it to `1` to see results directly in your terminal.
```

Comment:
i'm not sure if it is useful to add code and logic for this debug thing.

Just keep it super simple - add the console.log in your code.

If people want to remove them it is up to them.

Put the focus on things that matter. Having a debug argument does not matter and spread the focus on non important things.